scalafmt-tests/src/test/scala/org/scalafmt/FormatTests.scala

```diff
@@ -54,6 +54,13 @@ class FormatTests extends FunSuite with CanRunTests with FormatAssertions {
         e.partialOutput
       case x => x.get
     }
+    if (!onlyManual) {
```

Comment:
why did you reorder? i think the original idea was to put higher preference on idempotence violations, although I can't remember why. generally, if you believe this way is better, I'd strongly recommend doing that as part of a separate pull request, so the intent is clearly recorded in history.

Reply:
Not sure what @olafurpg meant here, but I guess I can remove the changes.

Reply:
Moved to https://github.com/scalameta/scalafmt/pull/2312